crates/driver/src/domain/competition/mod.rs

```diff
@@ -205,7 +213,13 @@ impl Competition {
 
         let id = settlement.id;
         *self.settlement.lock().unwrap() = Some(settlement);
-        Ok((id, score))
+
+        let reward = solution::Reward {
+            performance_address: self.solver.address(),
+            participation_address: self.solver.address(),
```

Comment:
I think right now every solver has a cow payout address that is different from the solver address itself.

Reply:
Same as here: https://github.com/cowprotocol/services/pull/1425#discussion_r1198807045